enterprise/kernel/src/test/java/org/neo4j/util/TestHelpers.java

```diff
@@ -61,6 +62,12 @@ public static Exception executionIsExpectedToFail( Runnable runnable )
     }
 
     public static int runBackupToolFromOtherJvmToGetExitCode( File neo4jHome, String... args ) throws Exception
+    {
+        return runBackupToolFromOtherJvmToGetExitCode( neo4jHome, new AtomicReference<>(), new AtomicReference<>(), true, args );
+    }
+
+    public static int runBackupToolFromOtherJvmToGetExitCode( File neo4jHome, AtomicReference<String> output, AtomicReference<String> errorOutput,
```

Comment:
Questionable solution with AtomicReference hack, feedback encourraged

Reply:
Is it not possible to just redirect the out and error streams?

Reply:
I think you can do getInputStream on the Process

Reply:
Good suggestion, I have replaced with PrintStream and it does come across better, thanks!